client/my-sites/media-library/content.jsx

```diff
@@ -136,11 +144,19 @@ module.exports = React.createClass( {
 					);
 					break;
 				default:
-					message = this.translate(
```

Comment:
:) I suppose we can duplicate storage in both Flux and Redux, if you think it'd be worthwhile. Yes, many of the MediaUtil methods would be good candidates to move into Redux Selectors.